chowpronto_frontend/src/App.js

```diff
@@ -7,27 +7,34 @@ import RegisterPage from "./patron/pages/register";
 import LandingPage from "./patron/pages/landing";
 import DeliverySelect from "./patron/components/DeliverySelect";
 import Theme from "./style/Theme";
+import { reducer } from "./state/reducer";
+import { initialState } from "./state/initialState";
+import { MenuContext } from "./state/MenuContext";
+import Context from "./state/Context";
```

Comment:
Only **Context** is being used here.  We'll need to remove the unused imports before merging

Reply:
Removed all unnecessary imports (apart from a few in useAuth which are being used in testing)